src/chrome/content/rules/Curse.xml

```diff
@@ -3,36 +3,187 @@
 
 		- bukkit.org.xml
 
+	TLS not supported on:
+	- forums.wowace.com (only supported by cloudflare frontend, not curse backend)
```

Comment:
Can you explain what exactly the issue is? They even redirect server side to https.